src/DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -658,3 +667,17 @@ function AnyEntryMapFixes()
     }
 }
 //#endregion
+
+//#region Timer
+function TimerMapFixes()
+{
+    switch(dxr.localURL)
+    {
+    case "02_NYC_BAR":
+        if (dxr.flagBase.GetBool('JaneyThankful_Played')) {
+            player().StartDataLinkTransmission("DL_ReminderJaney");
```

Comment:
> [SPEECH: AlexJacobson]
> Hey, JC.  I applaud your instincts here; just don't forget about the generator when you're done.  Paul's still bringing up his team, so you've got a little time.

does this sound right when Janey just gave you the code to the warehouse ramp?

Reply:
It's also possible to go and do the warehouse, then come back and save Sandra afterwards, in which case this infolink doesn't really fit